apacheconfig/loader.py

```diff
@@ -11,14 +11,16 @@
 import tempfile
 
 from apacheconfig.error import ApacheConfigError
+from apacheconfig.host import LocalHost
 
 log = logging.getLogger(__name__)
 
 
 class ApacheConfigLoader(object):
 
-    def __init__(self, parser, debug=False, **options):
+    def __init__(self, parser, host=LocalHost(), debug=False, **options):
```

Comment:
Just an observation that `LocalHost()` as a default parameter would end up in a single `LocalHost` class instance for all instances of the `ApacheConfigLoader` object. That may (in theory) cause race conditions within the `LocalHost` object in the multi-threaded environment.

The alternative approach would be to either pass class (not instance) or default to `None` and instantiate `LocalHost` within `ApacheConfigLoader.__init__()`...

WDYT?